orttraining/orttraining/python/training/ortmodule/_graph_execution_manager.py

```diff
@@ -321,6 +321,14 @@ def _get_exported_model(self, input_schema: _ModelInputOutputSchemaType, *inputs
             # FlattenedModule needs _InputInfo to expand user input from *args to *args + **kwargs
             self._flattened_module._input_info = self._input_info
 
+            # Leverage cached model if available
+            cache_dir = os.getenv("ORT_CACHE_DIR", default=None)
```

Comment:
You could use the runtime options to record the environment variable if possible.